web/src/helpers/constants.js

```diff
@@ -41,7 +41,7 @@ const LANGUAGE_NAMES = {
   da: 'Dansk',
   de: 'Deutsch',
   el: 'Ελληνικά',
-  en: 'English',
+  'en-GB': 'English',
```

Comment:
This change is the only thing blocking this from my side as we don't have the other necessary changes for this to work.
(Fallback language, us version for formatting and verified British translations)

What do you think @tonypls?

Reply:
I'd quite prefer if we can avoid this, as it's used in a lot of different places and might cause bugs elsewhere if we're not consistent with the locale names :)

Reply:
Agreed, the long term goal should be to support all sub locales of English (GB, US, AU, etc) but for now I think we need to stick with just plain `en` as the language key.